public/css/profilePage.css

```diff
@@ -10,7 +10,9 @@
 
 
 #profileHead {
-    background-image: url('../img/coverPhoto.jpg');
+    background-image: url('https://images.pexels.com/photos/326316/pexels-photo-326316.jpeg?cs=srgb&dl=close-up-of-vintage-camera-on-white-background-326316.jpg&fm=jpg');
```

Comment:
Ms. Unisse advised us not to put background images to keep it clean and readable.

Reply:
Would be resolved by Marts!